config/bootstrap.sh

```diff
@@ -19,6 +19,7 @@ install sqlite3
 install python3
 install python3-dev
 install python3-pip
-install sqlite3
+#install sqlite3
+install postgresql-server-dev-all
```

Comment:
Could you doublecheck the provisioning code here? I couldn't get it to run out-of-the-box. (Needed to install `postgresql-9.3`, `libpq-dev`).

At the end, we should be able to run:
```bash
$ vagrant destroy
$ vagrant up
```

and have things work out of the box.